app/cypress/integration/ui/crapi_loadgenTest.js

```diff
@@ -0,0 +1,77 @@
+///<reference types="Cypress"/>
+describe('My First test suite', function()
+{
+    it('My First testcase',function()
+    {
+        cy.visit("http://35.225.176.150/");
```

Comment:
@raveenareddyn can we take this from environment variable instead of hard-coding?